waffle/onnx/onnxgraph.py

```diff
@@ -1,13 +1,70 @@
 from typing import List
-from waffle.onnx.node import Node
-
+from waffle import tensor
+from waffle.onnx.node import Node  
+import os
 
+WFLDBG = os.environ.get('WFLDBG')
+#edit spaces for 2
 class onnxGraph:
-  def __init__(self, list:List[Node]):
-    self.liniearized_graph = list
+    def __init__(self, linearized_list):
+        self.nodes = [Node(node.name, node.input, node.output, node.op_type, node.attributes, node.params) for node in linearized_list]
+        self.build_graph()
```

Comment:
These two should not be called here! and also these two shuld be refactored to a single function called hard_traverse() which will be called in the nn/__ init __.py compile() function!